src/AggieEnterpriseApi/Queries/DisplayDetailsGl.graphql

```diff
@@ -0,0 +1,51 @@
+query DisplayDetailsGl($segmentString: GlSegmentString!, $validateCVRs: Boolean = true, $entity: String!,
```

Comment:
@sprucely For the GL, you also want to get the details for Project, Program, and activity
(These will often just be default, but not always)